src/services/auth.js

```diff
@@ -35,3 +35,48 @@ export const googleLoginRequest = async data => {
   setAuthToken(token);
   return decoded;
 };
+
+export const forgotPasswordRequest = async data => {
+  try {
+    const response = await axios.post('api/users/reset', data);
+    return response;
+  } catch (e) {
+    return {
+      status: 400,
+      data: 'Bad email',
+    };
```

Comment:
Replace this on validation on backend side

Reply:
This is a problem with axios with Try Catch. It throws Error Exception when server sends error message